AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1827,6 +1827,7 @@ qanda.ai##div[class^="Picture__Protector-"]
 codedosa.com###wpcp-error-message
 !--- CSS ---
 on.od.ua#$#.footer_wrapper { padding-bottom: 0px !important; }
+olympics.com#$#.b2p-main { padding-bottom: 0px !important; }
```

Comment:
It does nothing on my device. Maybe better change `padding-top:` to `62px`?

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/173892349-e87e9ae1-845e-450d-b6e9-7180c9aa8f19.png)

</details>

Reply:
I agree, it should be like that
```
olympics.com#$#.b2p-main { margin-top: 0 !important; }
```